tools/calibration_scripts_system/run_all_parsers.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+import os, glob, subprocess
+import time
+
+def main():
+    for sub in os.listdir(os.getcwd()):
+        if os.path.isdir(sub):
+            os.chdir(sub)
+            # Checks if there are any cal files in Manufacturer Cal Files and do not
```

Comment:
It would be good to have a force option to override the files if necessary. See docopt for excellent argument parsing.